src/App.tsx

```diff
@@ -1,10 +1,77 @@
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, {
+  useCallback,
+  useEffect,
+  useState,
+} from 'react';
+import { getTodos } from './api/todos';
+import { Todo } from './types/Todo';
+import { Errors } from './types/Errors';
+import { Status } from './types/Status';
 import { UserWarning } from './UserWarning';
+import { TodoList } from './components/TodoList';
+import { Header } from './components/Header';
+import { Footer } from './components/Footer';
+import { ErrorNotification } from './components/ErrorNotification';
 
-const USER_ID = 0;
+const USER_ID = 12130;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [filterBy, setFilterBy] = useState(Status.all);
+  const [visibleTodos, setVisibleTodos] = useState(todos);
+  const [errorMessage, setErrorMessage] = useState<Errors | null>(null);
+
+  const onAllclick = useCallback(() => {
```

Comment:
Use CamelCase, fix all cases if exist 

```suggestion
  const onAllСlick = useCallback(() => {
```